iwyu_globals.cc

```diff
@@ -328,15 +327,16 @@ void InitGlobals(clang::SourceManager* sm,
   function_calls_full_use_cache = new FullUseCache;
   class_members_full_use_cache = new FullUseCache;
 
-  for (Each<HeaderSearchPath> it(&search_paths); !it.AtEnd(); ++it) {
-    const char* path_type_name
-        = (it->path_type == HeaderSearchPath::kSystemPath ? "system" : "user");
-    VERRS(6) << "Search path: " << it->path << " (" << path_type_name << ")\n";
+  for (const HeaderSearchPath& entry : search_paths) {
+    const char* path_type_name =
+        (entry.path_type == HeaderSearchPath::kSystemPath ? "system" : "user");
+    VERRS(6) << "Search path: " << entry.path << " (" << path_type_name
+             << ")\n";
```

Comment:
It's more a personal taste to keep parenthesis on the same line. There is no indication it is actually better.